dotcom-rendering/src/web/components/SubheadingBlockComponent.tsx

```diff
@@ -1,19 +1,36 @@
-import { unwrapHtml } from '../../model/unwrapHtml';
-import { RewrappedComponent } from './RewrappedComponent';
+import { jsx } from '@emotion/react';
+import { JSDOM } from 'jsdom';
+import type { ReactNode } from 'react';
+import { Fragment } from 'react';
 
 type Props = { html: string };
 
+const parseHtml = (html: string): DocumentFragment => JSDOM.fragment(html);
+
+function isElement(node: Node): node is Element {
+	return node.nodeType === 1;
+}
+
+const getAttrs = (node: Node): NamedNodeMap | undefined =>
+	isElement(node) ? node.attributes : undefined;
+
```

Comment:
These are re-used from the blockquote component right? Could we put them in a shared lib?